apps/a11y-tests/src/axe-sarif-converter/custom-rule-configurations.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { colorConfiguration } from './custom-rule-configurations/color-rule';
```

Comment:
I'd recommend making a file `custom-rule-configurations/index.ts` and exporting all the custom rules from there. Then importing in this file just looks like `import { all, the, things } from './custom-rule-configurations/index'`.

Reply:
Also never mind here due to temp code